### DISCUSSION TITLE
How to get started tailwind css?

### DISCUSSION THREAD

**Discussion by gitshadow1** [tailwindlabs/tailwindcss#8711] [Help] [answered]
Hi,
I am a newer in tailwind community..
I noticed tailwind cons and pros in several articles.
What is the fastest way to get started it?

**Comment by tofuwu-dev** [comment, marked as answer]
Hi, You can googe it and can find the way easily. :)

[Play CDN !](https://tailwindcss.com/docs/installation/play-cdn)

Tailwind community built a Javascript library. 
```bash
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>CDN</title>
    <script src="https://cdn.tailwindcss.com/"></script>
  </head>
  <body>
    <p class="text-2xl font-bold">
       Play CDN!
    </p>
  </body>
</html>
```
You can refer installation of tailwind docs. https://tailwindcss.com/docs/installation
The best way is to find it and see yourself... :)

**Comment by gitshadow1** [comment]
Thanks @haodev007  It was helpful for me to get started it... and thanks for your advice finally. 💯

**Reply by tofuwu-dev** [reply]
Happy to get it out! :D
